experimental/dds/tree2/src/core/rebase/changeRebaser.ts

```diff
@@ -68,7 +67,6 @@ export interface ChangeRebaser<TChangeset> {
 		isRollback: boolean,
 		// TODO: make the repair store mandatory when all usages of this method have repair data support.
 		repairStore?: ReadonlyRepairDataStore,
-		cache?: boolean,
```

Comment:
We should remove the cache API and the map from ModularChangeRebaser too.

Reply:
Sorry I undo those changes accidentaly.